lighthouse-core/gather/gather-runner.js

```diff
@@ -203,6 +203,10 @@ class GatherRunner {
       log.log('status', status);
       return driver.endNetworkCollect();
     }).then(networkRecords => {
+      if (driver.online && networkRecords[0].failed) {
+        log.error('statusEnd', networkRecords[0].localizedFailDescription);
+        return Promise.reject(new Error(networkRecords[0].localizedFailDescription));
```

Comment:
The request could've failed for other reasons than 'not resolved'. Maybe my PR title is a bit too specific.